ignite/engine/engine.py

```diff
@@ -125,6 +125,10 @@ def compute_mean_std(engine, batch):
     _state_dict_all_req_keys = ("epoch_length", "max_epochs")
     _state_dict_one_of_opt_keys = ("iteration", "epoch")
 
+    DEBUG_EVENTS = 1
```

Comment:
@puhuk one of the requirements is to be able to combine debug flags: https://github.com/pytorch/ignite/issues/1992#issuecomment-1075132649

Please also add tests for this feature